.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Other questions
+about: Governance of core libraries other than `base`; CLC policies and administration.
+title: ''
+labels: ''
```

Comment:
Perhaps automatically add "meta" label? Whatever the one we already have is.

Reply:
There are two possible labels: "meta" and "core libraries". I can split this template into two, if you feel it's worth doing. 

The purpose is not that much about automatic label assignment, but to make sure that people do read `PROPOSALS.md` and understand expectations.

Reply:
I do think meta- and other-core-library questions should be triaged, but I don't expect there to be an overwhelming volume.